docs/reference/ocm_add_componentversions.md

```diff
@@ -27,8 +27,8 @@ componentversions, componentversion, cv, components, component, comps, comp, c
       --lookup stringArray        repository name or spec for closure lookup fallback
   -O, --output string             output file for dry-run
   -R, --replace                   replace existing elements
-  -S, --scheme string             schema version (default "v2")
```

Comment:
Why was this removed in this PR? It looks like a different fix.

Reply:
because the AddFlags was missing....